tests/E2E/test/campaigns/common_scenarios/category.js

```diff
@@ -113,17 +114,20 @@ module.exports = {
       test('should search for category ', () => client.searchByValue(CategorySubMenu.search_input, CategorySubMenu.search_button, categoryData.name + date_time));
       test('should click on "Edit" action', () => {
         return promise
-          .then(() => client.clickOnAction(CategorySubMenu.update_button))
-          .then(() => client.getParamFromURL('id_category', 2000));
+          .then(() => client.scrollWaitForExistAndClick(CategorySubMenu.update_button))
+          .then(() => client.getURL(2000))
+          .then((res) => global.param['id_category'] = res.value.split('categories/')[1].split("/")[0]);
```

Comment:
Same here, use regex instead?

Reply:
Good suggestion.
Now, we are quick fixing tests, we will do it in the next step : refactoring

Reply:
Replace it with regex